edb/edgeql/declarative.py

```diff
@@ -189,12 +189,44 @@ def get_fq_name(
         if extra_name:
             fq_name = s_name.QualName(
                 module=fq_name.module,
-                name=f'{fq_name.name}:{extra_name}',
+                name=f'{fq_name.name}@@{extra_name}',
             )
 
         return name, fq_name
 
 
+def get_verbosename_from_fqname(
```

Comment:
Why not use `get_verbosename_static()`?

Reply:
I'm not sure it would be compatible with the way tracer names things.